pythonforandroid/recipes/groestlcoin_hash/__init__.py

```diff
@@ -0,0 +1,27 @@
+from pythonforandroid.recipe import CythonRecipe
+import os
+
+
+class groestlcoin_hashRecipe(CythonRecipe):
+    version = '1.0.1'
+    url = 'https://pypi.python.org/packages/source/g/groestlcoin_hash/groestlcoin_hash-{version}.tar.gz'  # noqa
+    depends = ['python3crystax']
+    call_hostpython_via_targetpython = True
+    cythonize = False
+
+    def get_recipe_env(self, arch):
+        # Overriding the original method because we need to add the
+        # libpythonX.Y location to LDFLAGS to be able to link against it
+        # maybe this should be done in the super method and benefit
+        # other recipes instead?
+        env = super(groestlcoin_hashRecipe, self).get_recipe_env(arch)
+        ndk_dir = self.ctx.ndk_dir
+        python_version = '.'.join(self.ctx.python_recipe.version.split('.')[:2])
+        ndk_path = os.path.join(
+            ndk_dir, 'sources', 'python', python_version, 'libs', arch.arch
+        )
+        env['LDFLAGS'] = ' -L {}'.format(ndk_path) + env['LDFLAGS']
```

Comment:
This change makes sense, but I don't understand why it wouldn't be needed by other CythonRecipes.

Reply:
Agree, in fact I've been adding it in all my recipes e.g. https://github.com/AndreMiras/EtherollApp/blob/v20180517/src/python-for-android/recipes/scrypt/__init__.py#L26
That and also maybe the `-I` header path like here https://github.com/kivy/python-for-android/pull/1250/files#diff-569e13021e33ced8b54385f55b49cbe6

Reply:
The include dir is already included by CythonRecipe, at https://github.com/kivy/python-for-android/blob/master/pythonforandroid/recipe.py#L1045